### DISCUSSION TITLE
[Q&A] 请问如何配置Dall E 3生成图片的尺寸？

### DISCUSSION THREAD

**Discussion by huelsgp27** [lobehub/lobehub#1132] [General Question] [answered]
在设置中，没有看到配置插件的入口
![image](https://github.com/lobehub/lobe-chat/assets/117451375/c1246e43-bb58-40ca-b4d7-8079d6fea72f)

**Comment by arvinxx** [comment, marked as answer]
后续我们增加一下相应的设置
